### ISSUE TITLE
policy·value loss function bug

### ISSUE THREAD

**Issue by sabina381** [sabina381/tic-tac-toe#23]
`net.py`의 두 head 출력 형태:
```
# net.py
self.policy_head = nn.Sequential(
    ...,
    nn.Softmax(dim=1)   # 행동 개수만큼의 "확률분포"
)
self.value_head = nn.Sequential(
    ...,
    nn.Tanh()           # [-1, 1] 사이 "스칼라" 1개
)
```
그런데 `train_network.py`의 `_loss_function`에서 이렇게 loss function이 적용되어있음.
```
mse = F.mse_loss(pred_policy, y_policy)              # 확률분포에 MSE
cross_entropy = self.cross_entropy(pred_value, y_value)  # 스칼라에 CrossEntropy
````
원 논문에 따르면 policy에 CrossEntropy, value에 MSE를 적용해야 함.

**Comment by sabina381** [comment]
수정 완료, 확인 후 close 예정
